cmd/bootstrap/version/version.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
```

Comment:
could you fix this to match other files - we just relicensed

Reply:
Done.